Lib/asyncio/mixins.py

```diff
@@ -0,0 +1,23 @@
+"""Event loop mixins."""
+
+__all__ = ('LoopBoundedMixin',)
+
+import threading
+from . import events
+
+global_lock = threading.Lock()
+
+
+class LoopBoundedMixin:
```

Comment:
Please start the nam
```suggestion
class _LoopBoundedMixin:
```
The class is private. We already have things like `_SelectorSocketTransport` in asyncio.

Reply:
Fixed